test/Turn-test.js

```diff
@@ -4,10 +4,17 @@ const expect = chai.expect;
 const Turn = require('../src/Turn');
 const Card = require('../src/Card');
 
+// const card = new Card();
```

Comment:
Saving this so I can continue to circle on this.  I am trying to figure out how to use the beforeEach